src/plugins/dashboard/public/dashboard_app/url/sync_dashboard_url_state.ts

```diff
@@ -117,17 +117,11 @@ export const startSyncingExpandedPanelState = ({
 }) => {
   const expandedPanelSubscription = dashboardAPI?.expandedPanelId.subscribe(() => {
```

Comment:
When you subscribe to the `expandedPanelId` behaviour subject, the subscription should **provide** the value that was emitted - so you shouldn't need to call `dashboardAPI.expandedPanelId.value`. Instead, just get that value from the subscribe params :)

Reply:
OH! MINDBLOWN!!! [4234e10](https://github.com/elastic/kibana/pull/190086/commits/4234e10a3f4bbc2e935ca01f6c633c39a6606b7a)